src/pages/Search/SearchEditMultiple/SearchEditMultiplePage.tsx

```diff
@@ -0,0 +1,198 @@
+        if (transactionValues.length === 0) {
+            return activePolicyID;
+        }
+
+        const firstPolicyID = transactionValues[0]?.policyID;
+        const allSamePolicy = transactionValues.every((t) => t.policyID === firstPolicyID);
+
+        if (allSamePolicy && firstPolicyID) {
+            return firstPolicyID;
+        }
+
+        return activePolicyID;
+    }, [selectedTransactions, activePolicyID]);
+
+    const policy = policyID ? policies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`] : undefined;
+    const [policyTags] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID ?? '-1'}`, {canBeMissing: true});
+    const [policyCategories] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${policyID ?? '-1'}`, {canBeMissing: true});
+
+    const currency = policy?.outputCurrency ?? CONST.CURRENCY.USD;
+
+    useEffect(() => {
+        if (!draftTransaction) {
+            initBulkEditDraftTransaction(currency);
+        }
+
+        return () => {
+            clearBulkEditDraftTransaction();
+        };
+    }, []);
+
+    const save = () => {
+        if (!draftTransaction) {
+            return;
+        }
+
+        const changes: TransactionChanges = {};
+        if (draftTransaction.amount !== undefined && draftTransaction.amount !== 0) {
+            changes.amount = draftTransaction.amount;
+        }
+        if (draftTransaction.currency) {
+            changes.currency = draftTransaction.currency;
+        }
+        if (draftTransaction.merchant) {
+            changes.merchant = draftTransaction.merchant;
+        }
+        if (draftTransaction.comment?.comment) {
+            changes.comment = draftTransaction.comment.comment;
+        }
+        if (draftTransaction.created) {
+            changes.created = draftTransaction.created;
+        }
+        if (draftTransaction.category) {
+            changes.category = draftTransaction.category;
+        }
+        if (draftTransaction.tag) {
+            changes.tag = draftTransaction.tag;
+        }
+        if (draftTransaction.billable !== undefined) {
+            changes.billable = draftTransaction.billable;
+        }
+        if (draftTransaction.reimbursable !== undefined) {
+            changes.reimbursable = draftTransaction.reimbursable;
+        }
+
+        if (Object.keys(changes).length === 0) {
+            Navigation.dismissModal();
+            return;
+        }
+
+        updateMultipleMoneyRequests(selectedTransactionIDs, changes, policy, policyTags, policyCategories);
+
+        Navigation.dismissModal();
+    };
+
+    const displayCurrency = draftTransaction?.currency ?? currency;
+
+    const attendeesTitle = useMemo(() => {
+        const attendees = draftTransaction?.comment?.attendees;
+        if (!Array.isArray(attendees) || attendees.length === 0) {
+            return '';
+        }
+        return attendees.map((item) => item?.displayName ?? item?.login).join(', ');
+    }, [draftTransaction?.comment?.attendees]);
```

Comment:
```suggestion
```

If we want to we can do a follow up improvement. The backend won't support the change in the current form.